src/redux/books/books.js

```diff
@@ -1,51 +1,46 @@
 import { v4 as uuidv4 } from 'uuid';
-import initialBookState from './initialbookstate';
+
+const API = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/3pY2SkyCSVzmuCrf1UYY/books/';
 
 // Actions
 const LOAD = 'bookstore/books/LOAD';
-const CREATE = 'bookstore/books/CREATE';
-const UPDATE = 'bookstore/books/UPDATE';
-const REMOVE = 'bookstore/books/REMOVE';
 
 // Reducer
-export default function reducer(state = initialBookState, action) {
-  let newBook;
+export default function reducer(state = {}, action) {
   switch (action.type) {
     case LOAD:
-      return state;
-    case CREATE:
-      newBook = {
-        ...action.book,
-        id: uuidv4(),
-      };
-      return [...state, newBook];
-    case UPDATE:
-      return state.map((book) => {
-        if (book.id === action.book.id) {
-          return action.book;
-        }
-        return book;
-      });
-    case REMOVE:
-      return state.filter((book) => book.id !== action.bookId);
+      return action.state;
     default:
       return state;
   }
 }
 
 // Action Creators
-export function loadBooks() {
-  return { type: LOAD };
-}
+export const loadBooks = () => async (dispatch) => {
+  const response = await fetch(API);
+  const state = await response.json();
+  dispatch({ type: LOAD, state });
+};
 
-export function createBook(book) {
-  return { type: CREATE, book };
-}
+export const createBook = (book) => async (dispatch) => {
+  await fetch(API, {
+    method: 'POST',
+    body: new URLSearchParams({
+      item_id: uuidv4(),
+      title: book.title,
+      category: book.category,
+    }),
```

Comment:
- [ ] When I try to create a new book. The list shows:

![image](https://user-images.githubusercontent.com/79876959/145302868-aece11b8-19c5-4141-891a-04ac36efbe91.png)

You want to consider fixing the values of title and category keys in the body of your request. 😉